### DISCUSSION TITLE
Problem when using a multi-page base pdf.

### DISCUSSION THREAD

**Discussion by peterfigueiredo** [pdfme/pdfme#92] [Q&A]
Hi there,

I need a little bit of help. 

I am using pdfme for a while now for a local nodeJS app I have, and honestly I really like it. I use it mostly for reporting.
I have been asked to update my reports, and for the first time, I will need more than one page (2 in this case).
I basically use a word document (I need to keep the head and footer of the doc for the report) and I export it as pdf.
I use this pdf as my basePDF. When I try to generate my pdf (pdfme.generate({ template, inputs })), I get lots of errors on my terminal. This doesnt happen if my pdf is one page though. 

Even more weird, when I use the template design, it works OK in the browser, but if I download the template (or the code), I get exactly the same problem when executing locally.

Any ideia why this is happenning? I even have to brute force the stopping of the PDF generation.

Best,
Peter
![error](https://user-images.githubusercontent.com/37262279/185374775-0e2f6838-3e43-46f6-9313-b5ecfd937aa2.png)

**Comment by peterfigueiredo** [comment]
Any help would be greatly appreciated.

**Reply by hand-dot** [reply]
Hi @peterfigueiredo 
I'm sorry for the late reply...

Can you give me a minimal reproducible example?
I'd like to watch the template and input data.
